Change the user used by fdm if run as root to 'debianfdm' instead of '_fdm'
which is not allowed in debian. (Actually I don't quite remember if it's
strictly not permitted by debian's policy, but I've stopped caring.)

Index: pkg-fdm/MANUAL
===================================================================
--- pkg-fdm.orig/MANUAL
+++ pkg-fdm/MANUAL
@@ -92,11 +92,11 @@ installation location:
 	$ ./configure --prefix=/opt/fdm && make
 	$ sudo make install
 
-If being run as root, fdm requires a user named "_fdm" to exist. It will drop
+If being run as root, fdm requires a user named "debianfdm" to exist. It will drop
 privileges to this user and its primary group. The user may be added on
 OpenBSD with, for example:
 
-	# useradd -u 999 -s /bin/nologin -d /var/empty -g=uid _fdm
+	# useradd -u 999 -s /bin/nologin -d /var/empty -g=uid debianfdm
 
 It is not necessary to add a user if fdm is always started by a non-root user.
 
Index: pkg-fdm/fdm.h
===================================================================
--- pkg-fdm.orig/fdm.h
+++ pkg-fdm/fdm.h
@@ -52,7 +52,7 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 
-#define CHILDUSER	"_fdm"
+#define CHILDUSER	"debianfdm"
 #define CONFFILE	".fdm.conf"
 #define LOCKFILE	".fdm.lock"
 #define DEFLOCKTIMEOUT	10
